scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -100,65 +100,72 @@ namespace <%= namespace %>.SDK.iOS {
         }
 
         private static UpdateRequestStatus GetUpdateRequestStatusFromCheckoutUserErrors(List<UserError> errors) {
-
             var payErrors = new List<ApplePayError>();
+            var statusToReturn = ApplePayAuthorizationStatus.Failure;
 
             // Check to see if any of the user errors are not billing address errors
             // If it is not a billing address error return a failure
             foreach (var error in errors) {
                 var fields = error.field();
-                if (fields.Contains("billingAddress")) {
-                    string description = error.message();
+                var fieldsSet = new HashSet<string>(fields);
 
-                    try {
-                        string lastField = fields[fields.Count - 1];
-                        var payError = new ApplePayBillingAddressInvalidError(description, lastField);
-                        payErrors.Add(payError);
-                    } catch {
-                        return new UpdateRequestStatus(ApplePayAuthorizationStatus.Failure);
+                try {
+                    string lastField = fields[fields.Count - 1];
```

Comment:
Since we removed the `Contains` statement, how do we know that `fields.Count` is not 0?

Reply:
We don't, but the Try statement will catch it, in which case it is handled as an error